Mlem/App/Views/Pages/Instance/InstanceView.swift

```diff
@@ -130,20 +133,24 @@ struct InstanceView: View {
     
     @ViewBuilder
     func administrationTab(instance: any Instance) -> some View {
-        VStack(spacing: Constants.main.halfSpacing) {
+        VStack(spacing: Constants.main.standardSpacing) {
             ModlogButtonView(community: nil)
-            ForEach(instance.administrators_ ?? []) { person in
-                PersonListRow(person)
+            
+            VStack(spacing: Constants.main.halfSpacing) {
+                ForEach(instance.administrators_ ?? []) { person in
+                    PersonListRow(person)
+                }
             }
             
             if appState.firstApi.isAdmin {
-                Button("Add Administrator", systemImage: Icons.add) {
-                    navigation.openSheet(.personPicker(filter: .local) { person in
-                        Task { await addAdmin(person, added: true) }
-                    })
-                }
-                .buttonStyle(.capsule)
-                .padding(.top, Constants.main.halfSpacing)
+                Button("Add Administrator", systemImage: Icons.add, action: openAddAdminSheet)
+                    .buttonStyle(.capsule)
+                    .padding(.bottom, Constants.main.halfSpacing)
+                    .confirmationDialog("", isPresented: $showingConfirmation) {
+                        Button("Yes", action: addNewAdmin)
+                    } message: {
+                        Text("Really appoint \(newAdmin?.displayName ?? "this user") as administrator of \(instance.displayName)?")
```

Comment:
For localization we should avoid unwrapping to a fallback in this way because it might not work with all languages

```suggestion
    if let displayName = newAdmin?.displayName {
        Text("Really appoint \(displayName) as an administrator of \(instance.displayName)?")
    } else {
        Text("Really appoint this user as an administrator of \(instance.displayName)?")
    }
```